test/listener_test.py

```diff
@@ -216,6 +218,42 @@ def test_reader_writer(self):
             self.assertEqual(msg1, msg2)
             self.assertAlmostEqual(msg1.timestamp, msg2.timestamp)
 
+class canutilsLog(unittest.TestCase):
+
+    def test_reader_writer(self):
+        f = tempfile.NamedTemporaryFile('w', delete=False)
+        f.close()
+        filename = f.name
+        writer = can.canutilsLogWriter(filename)
+
+        for msg in TEST_MESSAGES:
+            writer(msg)
+        writer.stop()
+
+        messages = list(can.canutilsLogReader(filename))
+        self.assertEqual(len(messages), len(TEST_MESSAGES))
+        for msg1, msg2 in zip(messages, TEST_MESSAGES):
+            self.assertEqual(msg1, msg2)
+            self.assertAlmostEqual(msg1.timestamp, msg2.timestamp)
+
+class ascFileFormat(unittest.TestCase):
+
+    def test_reader_writer(self):
```

Comment:
This test looks good. If you have them, would you mind also including a `.asc` file sourced using another tool like Vector CANalyzer or CANoe and add it to e.g., `test/data/vectorlogfile.asc` etc?